config/config.exs

```diff
@@ -34,29 +34,30 @@ config :logger, :console,
 
 config :ueberauth, Ueberauth,
   providers: [
-    google:
-      {Ueberauth.Strategy.Google,
-       [
-         approval_prompt: "force",
-         access_type: "offline",
-         default_scope: "email profile",
-         hd: System.get_env("HOSTED_DOMAIN")
-       ]}
+    google: {
+      Ueberauth.Strategy.Google,
+      [
+        approval_prompt: "force",
+        access_type: "offline",
+        default_scope: "email profile",
+        hd: System.get_env("HOSTED_DOMAIN")
+      ]
+    }
   ]
 
 config :ueberauth, Ueberauth.Strategy.Google.OAuth,
   client_id: System.get_env("GOOGLE_CLIENT_ID"),
   client_secret: System.get_env("GOOGLE_CLIENT_SECRET")
 
+# optional
```

Comment:
Interesting. These look like changes due to using an older version of the autoformatter than the one I've recently used. I'm planning to remove comments like these in an upcoming commit.